cms/templates/js/course-video-settings.underscore

```diff
@@ -11,37 +11,13 @@
     <div class='course-video-settings-wrapper'>
         <div class='course-video-settings-message-wrapper'></div>
         <span class="course-video-settings-title"><%- gettext('Transcript Settings') %></span>
-        <div class='transcript-preferance-wrapper transcript-provider-wrapper'>
-            <label class='transcript-preferance-label' for='transcript-provider'><%- gettext('Transcript Provider') %><span class='error-icon' aria-hidden="true"></span></label>
-            <div class='transcript-provider-group' id='transcript-provider'></div>
-            <span class='error-info' aria-hidden="true"></span>
-        </div>
-        <div class='transcript-preferance-wrapper transcript-turnaround-wrapper'>
-            <label class='transcript-preferance-label' for='transcript-turnaround'><%- gettext('Transcript Turnaround') %><span class='error-icon' aria-hidden="true"></span></label>
-            <select id='transcript-turnaround' class='transcript-turnaround'></select>
-            <span class='error-info' aria-hidden="true"></span>
-        </div>
-        <div class='transcript-preferance-wrapper transcript-fidelity-wrapper'>
-            <label class='transcript-preferance-label' for='transcript-fidelity'><%- gettext('Transcript Fidelity') %><span class='error-icon' aria-hidden="true"></span></label>
-            <select id='transcript-fidelity' class='transcript-fidelity'></select>
-            <span class='error-info' aria-hidden="true"></span>
-        </div>
-        <div class='transcript-preferance-wrapper transcript-languages-wrapper'>
-            <span class='transcript-preferance-label'><%- gettext('Transcript Languages') %></span>
-            <span class='error-icon' aria-hidden="true"></span>
-            <div class='transcript-languages-container'>
-                <div class='languages-container'></div>
-                <div class="transcript-language-menu-container">
-                    <select class="transcript-language-menu" id="transcript-language" aria-labelledby="transcript-language-none"></select>
-                    <div class="add-language-action">
-                        <button class="button-link action-add-language"><%- gettext('Add') %><span class="sr"><%- gettext('Press Add to language') %></span></button>
-                        <span class="error-info" aria-hidden="true"></span>
-                    </div>
-                </div>
-            </div>
+        <div class='course-video-settings-content'>
+            <div class='transcript-preferance-wrapper transcript-provider-wrapper'></div>
+            <div class='course-video-transcript-preferances-wrapper'></div>
+            <div class='organization-credentials-wrapper'></div>
         </div>
         <div class='course-video-settings-footer'>
```

Comment:
What about moving this div into `course-video-transcript-preferences.underscore`?